### PR TITLE
Pulsar deployment

### PR DESCRIPTION
**Pull request by mira-miracoli**
CHANGED:
- new [HTCondor Role](https://github.com/grycap/ansible-role-htcondor) thanks to @LuigiMansi1
- Fixed NFS thanks to @LuigiMansi1
- `volume_v2` → `volume_v3`
- switched from `data` to `resource` blocks for everything except `external` network
- changed default values / names
- updated README

REMOVED:
- `pre_tasks.tf` → user should create a fully separate set of resources for the Pulsar endpoint (except for public net)

### CODE REVIEW

File: tf/vars.tf [hunk 1]
```diff
@@ -24,29 +26,29 @@ variable "gpu_node_count" {
 }
 
 variable "image" {
-  type = map
+  type = map(any)
   default = {
-    "name"             = "vggp-v60-j225-1a1df01ec8f3-dev"
-    "image_source_url" = "https://usegalaxy.eu/static/vgcn/vggp-v60-j225-1a1df01ec8f3-dev.raw"
+    "name"             = "<name for that image>"
```
Comment (sebastian-luna-valero) [inline tf/vars.tf:32]:
```suggestion
//  "name"             = "vggp-v60-j225-1a1df01ec8f3-dev"
    "name"             = "<name for that image>"
```

File: tf/vars.tf [hunk 2]
```diff
@@ -24,29 +26,29 @@ variable "gpu_node_count" {
...
+    "image_source_url" = "<url-to-latest-vgcn-image>"
```
Comment (sebastian-luna-valero) [inline tf/vars.tf:34]:
```suggestion
//  "image_source_url" = "https://usegalaxy.eu/static/vgcn/vggp-v60-j225-1a1df01ec8f3-dev.raw"
    "image_source_url" = "<url-to-latest-vgcn-image>"
```

File: tf/vars.tf [hunk 3]
```diff
@@ -24,29 +26,29 @@ variable "gpu_node_count" {
...
   }
 }
 
 variable "gpu_image" {
-  type = map
+  type = map(any)
   default = {
-    "name"             = "vggp-gpu-v60-j16-4b8cbb05c6db-dev"
-    "image_source_url" = "https://usegalaxy.eu/static/vgcn/vggp-gpu-v60-j16-4b8cbb05c6db-dev.raw"
+    "name"             = "<name for that gpu image>"
```
Comment (sebastian-luna-valero) [inline tf/vars.tf:45]:
```suggestion
//  "name"             = "vggp-gpu-v60-j16-4b8cbb05c6db-dev"
    "name"             = "<name for that gpu image>"
```

File: tf/vars.tf [hunk 4]
```diff
@@ -24,29 +26,29 @@ variable "gpu_node_count" {
...
+    "image_source_url" = "<url-to-latest-vgcn-gpu-image>"
```
Comment (sebastian-luna-valero) [inline tf/vars.tf:47]:
```suggestion
//  "image_source_url" = "https://usegalaxy.eu/static/vgcn/vggp-gpu-v60-j16-4b8cbb05c6db-dev.raw"
    "image_source_url" = "<url-to-latest-vgcn-gpu-image>"
```

**Review by sebastian-luna-valero — COMMENTED** [review]
Thanks for this!

Minor comments from my side, I would leave comments as examples as shown below

**Comment by mira-miracoli** [comment]
yes thank you @sebastian-luna-valero I have overseen these

**Pull request merged**